gears/gears-kotlin/build.gradle.kts

```
@@ -0,0 +1,10 @@
+plugins {
+    convention.library.kotlin
+}
+
+version = "1.0.0"
```

And here, let's start from 0.1.0
```suggestion
version = "0.1.0"
```